sdk/schemaregistry/schema-registry/package.json

```
@@ -0,0 +1,124 @@
+{
+  "name": "@azure/schema-registry",
+  "version": "0.1.0",
```

I wasn't sure what version to put here.

---

```suggestion
  "version": "1.0.0-preview.1",
```

this is traditional, though we are thinking of changing from `preview` to `prerelease`

---

We can go with `1.0.0-preview.1` for now. This package will take another month to ship, we can change later to prerelease if needed

---

Done.